BurnOutSharp/PackerType/PECompact.cs

```diff
@@ -30,9 +30,13 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             return MatchUtil.GetFirstMatch(file, fileContent, matchers, includePosition);
         }
 
+        // TODO: Improve version detection, Protection ID is able to detect ranges of versions. For example, 1.66-1.84 or 2.20-3.02.
         public static string GetVersion(string file, byte[] fileContent, List<int> positions)
         {
-            return $"v{BitConverter.ToInt16(fileContent, positions[0] + 4)}";
+            short version = BitConverter.ToInt16(fileContent, positions[0] + 4);
+            if (version == 0)
+                return null;
```

Comment:
To quote the code in MatchUtil: `// A null version returned means the check didn't pass at the version step`

Reply:
If you can just change this to `string.Empty` instead, I'd appreciate it.